bin/yaml/cpp.yaml

```diff
@@ -681,6 +681,18 @@ compilers:
       check_exe: bin/tcc -v
       targets:
         - 0.9.27
+    cl430:
+      type: script
+      dir: "ti-cgt-msp430_{name}"
+      check_exe: bin/cl430 -version
+      targets:
+        - name: 21.6.1.LTS
+          fetch:
+            - https://dr-download.ti.com/software-development/ide-configuration-compiler-or-debugger/MD-p4jWEYpR8n/21.6.1.LTS/ti_cgt_msp430_21.6.1.LTS_linux-x64_installer.bin installer.bin
+          script: |
+            chmod +x installer.bin
+            ./installer.bin --unattendedmodeui none --mode unattended --installdir .
```

Comment:
When I ran this locally it created a directory in my home directory called `ti-cgt-msp430_21.6.1.LTS`. Nothing got installed in the path supplied to `--installdir`

Running `--help` seems to imply there isn't a `--installdir` option. There's a `--prefix` which if I pass it installs there:
```
~ $ ./installer.bin --unattendedmodeui none --mode unattended --prefix /tmp/moo/moo
~ 3.7s $ ls -ltr /tmp/moo/moo
total 4
drwxrwxr-x 6 matthew matthew 4096 Jan  3 07:43 ti-cgt-msp430_21.6.1.LTS
```

Reply:
> I'm going to try and make the relevant changs locally and merge, in the spirit of moving things forward.

Thanks. I'm not sure why installdir seemed to work for me, maybe I had it installed already and forgot to remove it. There are some other TI installers which use `--prefix`, so that's probably the correct option here too.